lib/deck.rb

```diff
@@ -10,22 +10,21 @@ def add_card(card)
   end
 
   def rank_of_card_at(index)
-    @cards[index].rank
+    cards[index].rank
   end
 
   def high_ranking_cards
-    highest = @cards.find_all do |card|
```

Comment:
`find_all` returns the found objects, so setting it equal to `highest` and then returning `highest` is redundant.